src/Take.Blip.Builder/Hosting/ConventionsConfiguration.cs

```diff
@@ -30,7 +30,9 @@ public class ConventionsConfiguration : IConfiguration
 
         public int ExecuteScriptMaxStatements => 1000;
 
-        public long ExecuteScriptLimitMemory => 65536;
+        public long ExecuteScriptLimitMemory => 0x100000; // 1MiB or 1 << 20
```

Comment:
Use decimal representation:
```suggestion
        public long ExecuteScriptLimitMemory => 1_000_000;
```